data/hai ships.txt

```diff
@@ -45,7 +45,7 @@ ship "Aphid"
 	explode "tiny explosion" 10
 	explode "small explosion" 20
 	explode "medium explosion" 15
-	description "The Aphid is a light freighter designed by the Hai. It has only minimal armament."
+	description "The Aphid is the lightest ship designed by the Hai. Similar to the human Star Barge, it can carry only a single turret for defense in case of emergencies, and has excellent cargo capacity for it's size."
```

Comment:
The Aphid isn't the lightest, at least not after the middle of the Wanderer plots (and the data file implies an even lighter ship in the future.)

"The Aphid is the lightest ship designed by the Hai that is capable of inter-system travel." or "The Aphid is the lightest ship designed by the Hai that comes equipped with a hyperdrive."

I'd lean towards something like "Similar to the human Star Barge, it attempts to discourage piracy with a single weapon turret, ..."